projects/igniteui-angular/migrations/common/tsUtils.ts

```diff
@@ -339,6 +358,48 @@ export const isMemberIgniteUI =
             && change.definedIn.indexOf(typeDef.name) !== -1;
     };
 
+const resolveMemberInfo = (langServ: tss.LanguageService, node: tss.Node): MemberInfo | null => {
+    const typeChecker = langServ.getProgram().getTypeChecker();
+    const nodeType = typeChecker.getTypeAtLocation(node);
```

Comment:
Any chance this will return something `getTypeArguments` won't like? The errorType?

Reply:
If you try to call it on a token, for example, It will return an intrinsic type (`error`) and then `getTypeArguments` will return an empty array. So it shouldn't throw.